### PR TITLE
Update page in Connect AWS Gateway “deploy-your-agents-1”

### PR DESCRIPTION
**Pull request by sbolosan**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by sbolosan** [comment]
doc ticket -> https://jira.axway.com/browse/APIGOV-16154

**Comment by sbolosan** [comment]
@dfeldick @kflan-io @jcollins-axway @vivekschauhan please review.  Updating doc for "approval" issues that jason found

**Review by jcollins-axway — APPROVED** [review]

**Comment by jcollins-axway** [comment]
What about line 99 and below, the examples for the different smtp servers

**Review by dfeldick — APPROVED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes

**Comment by sbolosan** [comment]
@jcollins-axway , yes.. i need to fix those as well... thanks.

**Review by kflan-io — APPROVED** [review]

**Review by kflan-io — APPROVED** [review]

**Comment by sbolosan** [comment]
@jcollins-axway @dfeldick @kflan-io @vivekschauhan please review based on Jason's findings in code snippet missing _NOTIFICATIONS . Thanks

File: content/en/docs/central/connect-aws-gateway/deploy-your-agents-1.md [hunk 1]
```diff
@@ -103,25 +103,25 @@ SUBSCRIPTIONS_SMTP_PASSWORD=your GMAIL password
...
-SUBSCRIPTIONS_SMTP_HOST=smtp.office365.com
-SUBSCRIPTIONS_SMTP_PORT=587
-SUBSCRIPTIONS_SMTP_USERNAME=your Office Mail account
-SUBSCRIPTIONS_SMTP_PASSWORD=your Office Mail password
-SUBSCRIPTIONS_SMTP_AUTHTYPE=LOGIN
+SUBSCRIPTIONS_NOTIFICATIONS_SMTP_HOST=smtp.office365.com
+SUBSCRIPTIONS_NOTIFICATIONS_SMTP_PORT=587
+SUBSCRIPTIONS_NOTIFICATIONS_SMTP_USERNAME=your Office Mail account
+SUBSCRIPTIONS_NOTIFICATIONS_SMTP_PASSWORD=your Office Mail password
+SUBSCRIPTIONS_NOTIFICATIONS_SMTP_AUTHTYPE=LOGIN
```
Comment (jcollins-axway) [inline content/en/docs/central/connect-aws-gateway/deploy-your-agents-1.md:110]: CENTRAL_?

Comment (sbolosan) [reply content/en/docs/central/connect-aws-gateway/deploy-your-agents-1.md:110]: @jcollins-axway please review... if at first you don't succeed.

**Review by kflan-io — APPROVED** [review]

**Review by dfeldick — APPROVED** [review]

**Review by dfeldick — APPROVED** [review]

**Review by jcollins-axway — APPROVED** [review]

File: content/en/docs/central/connect-aws-gateway/deploy-your-agents-1.md [hunk 2]
```diff
@@ -103,11 +103,11 @@ CENTRAL_SUBSCRIPTIONS_SMTP_PASSWORD=your GMAIL password
...
 
 # Microsoft office server
-SUBSCRIPTIONS_NOTIFICATIONS_SMTP_HOST=smtp.office365.com
-SUBSCRIPTIONS_NOTIFICATIONS_SMTP_PORT=587
-SUBSCRIPTIONS_NOTIFICATIONS_SMTP_USERNAME=your Office Mail account
-SUBSCRIPTIONS_NOTIFICATIONS_SMTP_PASSWORD=your Office Mail password
-SUBSCRIPTIONS_NOTIFICATIONS_SMTP_AUTHTYPE=LOGIN
+CENTRAL_SUBSCRIPTIONS_NOTIFICATIONS_SMTP_HOST=smtp.office365.com
+CENTRAL_SUBSCRIPTIONS_NOTIFICATIONS_SMTP_PORT=587
+CENTRAL_SUBSCRIPTIONS_NOTIFICATIONS_SMTP_USERNAME=your Office Mail account
```
Comment (dfeldick) [inline content/en/docs/central/connect-aws-gateway/deploy-your-agents-1.md:108]: I missed this before. I think that if you are referring to MS office mail, it is formally Microsoft Office 365

Comment (sbolosan) [reply content/en/docs/central/connect-aws-gateway/deploy-your-agents-1.md:108]: @dfeldick  fixed... both Office and Outlook

**Review by cbordier — APPROVED** [review]
I approve the changes

**Review by dfeldick — APPROVED** [review]

**Pull request merged**